### PR TITLE
Improve text formatting in coala settings docs

### PR DESCRIPTION
**Pull request by Mariatta**
- use short url
- highlight the command line code

Fixes https://github.com/coala/coala/issues/3449

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: coalib/parsing/DefaultArgParser.py [hunk 1]
```diff
@@ -36,8 +36,7 @@ def default_arg_parser(formatter_class=None):
 code, regardless of the programming languages you use.
 
 To find out what kind of analysis coala offers for the languages you use, visit
-<https://github.com/coala/bear-docs/blob/master/README.rst#supported-languages>
-or run:
+http://coala.io/languages, or run::
```
Comment (sils) [inline coalib/parsing/DefaultArgParser.py]: don't wanna use enclosing `<>`?

**Comment by sils** [comment]
otherwise good

File: coalib/parsing/DefaultArgParser.py (hunk 1 above)
Comment (Mariatta) [inline coalib/parsing/DefaultArgParser.py]: The <> is not necessary and looks weird when viewed in the docs
screen shot 2017-01-01 at 11 00 25 pm

Comment (sils) [inline coalib/parsing/DefaultArgParser.py]: I stand corrected!

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3451/commits/87747ab4af98e335d9acfc719b0e012917572969

**Comment by sils** [comment]
@rultor merge

**Pull request merged**